README.md

```diff
@@ -1 +1,153 @@
+Depending on where the input data are coming from and output data are sending to, the workflow can be running in two modes: `Local` and `RDPC` . The major tasks performed in the workflow are:
+- (`RDPC` mode only) Download input sequencing metadata/data from data center using SONG/SCORE client tools
+- Perform `Bcftools view` to count indels and snps counts and ratios
+- Perform `Bcftools stats`  to collect stats for VCF
+- (`RDPC` mode only) Generate SONG metadata for all collected QC metrics files and upload QC files to SONG/SCORE
+
+
+### References
+- Reference genome: 
+  - GRCh38 reference genome fasta file. The file can be downloaded by:
+    ```bash
+    wget https://object.genomeinformatics.org/genomics-public-data/reference-genome/GRCh38_hla_decoy_ebv/GRCh38_hla_decoy_ebv.fa
+    ``` 
+
+  - GRCh38 reference genome fasta index file. The file can be downloaded by:
+    ```bash
+    wget https://object.genomeinformatics.org/genomics-public-data/reference-genome/GRCh38_hla_decoy_ebv/GRCh38_hla_decoy_ebv.fa.fai
+    ```
+
+- Autosome non-gap regions
+  - `autosome_non_gap` bed file was downloaded from [NPM-sample-qc](https://raw.githubusercontent.com/c-BIG/NPM-sample-qc/master/resources/autosomes_non_gap_regions.bed) and staged under project folder [assets](https://github.com/icgc-argo-workflows/dnaalnqc/tree/main/assets)
+
+> **NOTE**
+> Please stage the reference files into the reference directory <REFERENCE_BASE> with the following folder structure
+```bash
+<REFERENCE_BASE>
+├── GRCh38_hla_decoy_ebv.fa
+├── GRCh38_hla_decoy_ebv.fa.fai
+```
+
+
+### Inputs
+#### Local mode
+First, prepare a sample sheet with your input data that looks as following example:
+
+`sample_sheet.csv`:
+
+```csv
+sample,vcf
+HG00100,assets/test/HG00100.hard-filtered.vcf.gz
+HG00844,assets/test/HG00844.hard-filtered.vcf.gz
+HG03722,assets/test/HG03722.hard-filtered.vcf.gz
+```
+
+Each row represents an aligned VCF or BCF from a sample.
+
+Then, you need to download [Autosome non-gap regions](#references), and optionally [reference files](#references) staged in <REFERENCE_BASE>
+
+Now, you can run the workflow using:
+
+```bash
+nextflow run icgc-argo-workflows/dnaalnqc \
+   -profile <standard/singularity> \
+   --local_mode true \
+   --input sample_sheet.csv \
+   --fasta <REFERENCE_BASE>/GRCh38_hla_decoy_ebv.fa \
+   --fasta_fai <REFERENCE_BASE>/GRCh38_hla_decoy_ebv.fa.fai \
+   --regions assets/autosomes_non_gap_regions.bed \
+   --outdir <OUTDIR>
+```
+
+#### RDPC mode
+You can run the workflow in RDPC mode by using:
+```bash
+nextflow run icgc-argo-workflows/dnaalnqc \
+    -profile <rdpc,rdpc_qa,rdpc_dev>,<standard/singularity> \
+    --local_mode false \
+    --study_id <STUDY_ID> \
+    --analysis_ids <ANALYSIS_IDS> \
+    --api_token <YOUR_API_TOKEN> \ 
+    --fasta <REFERENCE_BASE>/GRCh38_hla_decoy_ebv.fa \
+    --fasta_fai <REFERENCE_BASE>/GRCh38_hla_decoy_ebv.fa.fai \
+    --regions assets/autosomes_non_gap_regions.bed \
+  --outdir <OUTDIR>
+```
```

Comment:
Please make changes accordingly for running `vcfqc`. Do we need the reference genome for running vcfqc?